### PR TITLE
Notes/GitHub and markdown

### PR DESCRIPTION
**Pull request by alhen002**
First i added a new file git-branches-pr.md file in the shell-and-git folder. After that i added more content to that file.

### CODE REVIEW

File: shell-and-git/git-branches-pr.md
Comment (UmaySoyluer) [inline shell-and-git/git-branches-pr.md:1]: wonderful content!!!super

File: shell-and-git/git-branches-pr.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# git branches & pull requests
+
+## git branches
+
+`git switch -c header` -> creates a new branch header and switch to it
+`git switch` -> switch to branch
```
Comment (UmaySoyluer) [inline shell-and-git/git-branches-pr.md]: please make it more descriptive.

**Review by UmaySoyluer — CHANGES_REQUESTED** [review]
please make changes asap:P

File: shell-and-git/git-branches-pr.md (hunk 1 above)
Comment (alhen002) [reply shell-and-git/git-branches-pr.md]: will be better in the next commit

**Review by UmaySoyluer — APPROVED** [review]

**Pull request merged**